### ISSUE TITLE
I can't seem to download the blogger/share <textarea/> code from https://www.downthemall.org/development/bloggers/spread-dta/

### ISSUE THREAD

**Issue by hourback** [downthemall/downthemall#274]
**Desktop (please complete the following information):**
 - OS: [e.g. Windows 10, macOS, Linux (distribution, desktop environment)]
 - Browser and version: [e.g. Firefox 42, Chrome 70, Opera 15, Seamonkey 2.16]
 - DownThemAll! version: [e.g. 4.2, 3.0, latest]

**Describe the bug**

**To Reproduce**

**Expected behavior**

**Screenshots**

**Additional context**

**Comment by d7415** [comment]
You're right - there seem to be 2 `<p>` tags sitting on top of them. I suspect they just need to be placed in their own `<div>`.
